source/_posts/2017-06-30-uefi-slash-edk-ii-corebootpkg-on-pc-engines-apu2.markdown

```diff
@@ -8,26 +8,37 @@ categories: coreboot uefi edk2 apu2
 ---
 
 Recently we were reached by person interested in running CoreOS on APU2. CoreOS
-is very interesting system from security point of view. It was created to
+is a very interesting system from security point of view. It was created to
 support containers and scalability out of the box. Unfortunately it requires firmware
 supporting GPT. At that point I was not sure if I can utilize GRUB GPT support
 on APU2, but this led to other questions:
 
 - Is it possible to boot UEFI-aware OS on PC Engines APUx boards ?
-- What level of security can I get with UEFI-awere OS in comparison to coreboot ?
+- What level of security can I get with UEFI-aware OS in comparison to coreboot ?
 
 Those questions were much more interesting to firmware developer, because of
 that we decided to triage [coreboot UEFI payload](https://github.com/tianocore/tianocore.github.io/wiki/Coreboot_UEFI_payload)
 on PC Engines APU2 platform.
 
 For interested ones in that topic I recommend to take look at [video from coreboot conference 2016](https://youtu.be/I08NHJLu6Us?list=PLiWdJ1SEk1_AfMNC6nD_BvUVCIsHq6f0u).
 
-All my modifications of the article below can be found in [3mdeb edk2 fork](https://github.com/3mdeb/edk2/tree/apu2-uefi)
+All my modifications of edk2 for the article below can be found in [3mdeb edk2 fork](https://github.com/3mdeb/edk2/tree/apu2-uefi)
 
 For those interested in UEFI-aware OS booting this blog post can be useful, but
 I also plan to write something straightforward that can be used and read by
 APUx platform users.
 
+**NOTE**: this blog post wait so long for publishing that coreboot community
+provided and improved support for tianocore payload. It can be choose from
```

Comment:
"chosen"

Reply:
@miczyg1 little bit late this blog post was already published. We are not using this repository any more.